src/metadataGeneration/tsoa.ts

```diff
@@ -62,6 +63,7 @@ export namespace Tsoa {
     default?: any;
     description?: string;
     format?: string;
+    example?: any;
```

Comment:
```suggestion
    example?: unknown;
```

Reply:
I went ahead and move the type over. Any => unknown is a good conversion in general